gcloud-java-examples/src/main/java/com/google/gcloud/examples/nio/Stat.java

```diff
@@ -20,6 +20,9 @@
  * be logged in (using e.g. the gcloud auth command).
  *
  * <p>See the README for a command line to run this example.
+ *
```

Comment:
I agree it's a bit odd, but the benefit would be that we can see that the very same code works using either mechanism.

If that bothers you, we can certainly replace it by something else - I'd prefer this to having the modules depend from each other. 

How about just something that lists the filesystem providers? We can call it "ListFilesystems". That still makes it super-easy to compare the output with and without the nio jar. It's a subset of what Stat can do but that should be enough to remove the confusion. What do you think?